### DISCUSSION TITLE
Animetrix Discussion

### DISCUSSION THREAD

**Discussion by ShivaBhattacharjee** [ShivaBhattacharjee/AnimeTrix#14] [Ideas]
Welcome to the Animetrix discussion board! Here, you can ask questions, share ideas, and discuss anything related to the project.

General Questions
If you have any general questions about Animetrix, feel free to ask them here. This could include questions about how to use the application, how the application works, or any other general inquiries.

Feature Requests
If you have any ideas for new features that you would like to see added to Animetrix, please share them here. Be as detailed as possible in your request, and provide any relevant information that you think might be useful.

Bug Reports
If you have encountered any bugs or issues while using Animetrix, please report them here. Be sure to provide as much information as possible about the issue, including steps to reproduce the problem, any error messages that you have seen, and any other relevant information.

Feedback
We would love to hear your feedback on Animetrix! If you have any suggestions, comments, or feedback, please feel free to share it here.

Other
If you have any other questions or comments that don't fit into the categories above, feel free to share them here.

Thank you for your interest in Animetrix, and we look forward to hearing from you!

**Comment by SoumyaEXE** [comment]
Hey @ShivaBhattacharjee ! It Will Be Great If you add a 10 sec skip in player so it will be more great. And what if you add a dub anime section and previous episode and next episode button.

Ignore My English Grammar Etc. 😅!

**Reply by ShivaBhattacharjee** [reply]
I am currently working on sub to dub option in animeTrix also theres a 5s skip in the player regarding next and previous button i will look into it

**Reply by SoumyaEXE** [reply]
Thanks!

**Comment by Yuzuru-Yamazaki** [comment]
It was kinda fast for another version

**Comment by Yuzuru-Yamazaki** [comment]
And I got an issue, why can't I deploy it on netlify or vercer?

**Reply by ShivaBhattacharjee** [reply]
Some packages might be deprecated can you provide me the error logs or go to settings and change the command from npm install to npm install --force

**Reply by Yuzuru-Yamazaki** [reply]
Did you see all the forks and stars ! Whoa

**Reply by ShivaBhattacharjee** [reply]
Yeah 100 star is the milestone

**Reply by Yuzuru-Yamazaki** [reply]
You deserve 1000x more

**Reply by Yuzuru-Yamazaki** [reply]
Zoro isn't open source but it is famous , thus your site should be as well

**Reply by Yuzuru-Yamazaki** [reply]
You should add a login system like Thor ,
hoshino-ai.vercel.app

**Reply by ShivaBhattacharjee** [reply]
> You should add a login system like Thor ,
> hoshino-ai.vercel.app

Theres already a login/sign up system in animeTrix 

![Screenshot_20230604-000159_Chrome.png](https://github.com/ShivaBhattacharjee/AnimeTrix/assets/95211406/fef82e5d-ac99-405c-9a94-668bc74fb238)

**Reply by Yuzuru-Yamazaki** [reply]
Whoa
Do you sign up with anilist?

**Reply by Yuzuru-Yamazaki** [reply]
I don't know where you got the server to contain user's information, but my Pfp sucks

**Reply by ShivaBhattacharjee** [reply]
> I don't know where you got the server to contain user's information, but my Pfp sucks

The server side code isnt open-source however i am using mongo db also sorry for the pfp you can't change it for the time being 💀

**Reply by Yuzuru-Yamazaki** [reply]
Bruh I'm stuck with it

**Reply by ShivaBhattacharjee** [reply]
> Bruh I'm stuck with it 

What happened?

**Reply by SoumyaEXE** [reply]
> And I got an issue, why can't I deploy it on netlify or vercer?

**FOR VERCEL USE :**

_yarn install_


And

_yarn build_
